### DISCUSSION TITLE
Profile displaying wrong number of repositories.

### DISCUSSION THREAD

**Discussion by LyonsHwang** [community/community#29809] [Other Feature Feedback, Questions, & Ideas]
There 2 repositories of mine but profile page displays a total 3. I have tried to create a new public repo to refresh but the difference is still there. How to correct it?

![image](https://user-images.githubusercontent.com/61009012/185148811-04e32c56-3ace-4ccd-b260-e57241caf268.png)

**Comment by jfullstackdev** [comment]
did you just make it private ? 

![image](https://user-images.githubusercontent.com/47092464/185151146-9c3666e1-b284-4930-ba22-0c9722ec1d5d.png)

I see 2 but it may be because you changed the other one from public to private

**Comment by arotsh** [comment]
@LyonsHwang I guess you make it private because for now, I can only see one repo but it shows 2. If you want to make it public you just need to go to your account -> click your private repo -> settings -> at the very bottom click change visibility -> make public -> write `LyonsHwang/YOUR_REPO_NAME` -> click `I understand, change repository visibility`

**Comment by LyonsHwang** [comment]
Now I have made all my repos public and it shows 3.

**Reply by LyonsHwang** [reply]
![X{76MPDT@22J)~KJOU KTT1](https://user-images.githubusercontent.com/61009012/185189820-9275a8f5-0b26-48a5-a6ca-5d7c32ac5ffa.png)
![JR`)WA_R)Q9TNWR_C1}GNMX](https://user-images.githubusercontent.com/61009012/185189925-81e09209-ddfc-463c-ad6c-c989238a0747.png)

**Reply by arotsh** [reply]
Okay now all repositories are public, can you pls try to create a new public repository and delete it, I think it will update to the correct number

**Reply by LyonsHwang** [reply]
It didn't work for me...
![I333$ZW`QO(}O 6J$FI)4PR](https://user-images.githubusercontent.com/61009012/185193957-2ebee6ed-929f-460f-8dcc-2c207337f43a.png)
![3 1 8Y2T9VEC$F}E~X~F23W](https://user-images.githubusercontent.com/61009012/185193988-dac0e68d-8714-45a3-8652-5a39d18a86d4.png)

**Reply by arotsh** [reply]
Oh, then the only thing that I can suggest make a report.

**Comment by VelpCode** [comment]
Hmm.. I'd suggest try making a few more blank repositories, wait it out for a while and see whats up. Thats a weird bug though

**Reply by LyonsHwang** [reply]
Ok, I will take a try.
![1660794391215](https://user-images.githubusercontent.com/61009012/185288933-0d0e8299-78ad-470b-9fbb-538f34dfe43e.png)
